File: 13Jan2024.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        fre = [0]*26
        for i in range(0,len(s)) :
            fre[ord(s[i])-ord('a')]+=1
            fre[ord(t[i])-ord('a')]-=1
        ans=0;
        for f in fre :
            if(f>0) :
                ans+=f
        return ans

File: 13Jan2024.java
class Solution {
    public int minSteps(String s, String t) {
        int[] fre = new int[26];
        for (int i = 0; i < s.length(); i++) {
            fre[s.charAt(i) - 'a']++;
            fre[t.charAt(i) - 'a']--;
        }
        int ans = 0;
        for (int f : fre)
            if (f > 0)
                ans += f;
        return ans;
    }
}